controllers/admin/AdminAjaxPrestashopCheckoutController.php

```diff
@@ -36,6 +36,16 @@ class AdminAjaxPrestashopCheckoutController extends ModuleAdminController
      */
     public $module;
 
+    /**
+     * @var bool
+     */
+    public $ajax = true;
+
+    /**
+     * @var bool
+     */
+    protected $json = true;
```

Comment:
We have an issue with this property when we try to switch mode sandbox/live. I've commented it the branch v3.0.0 and it seems working well.